ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -213,7 +213,7 @@ def __init__(self, vm_names, vm_properties, fp_mtu, max_fp_num, topo):
         self._host_interfaces_active_active = None
         return
 
-    def init(self, vm_set_name, vm_base, duts_fp_ports, duts_name, ptf_exists=True):
+    def init(self, vm_set_name, vm_base, duts_fp_ports, duts_name, ptf_exists=True, check_bridge=True):
         self.vm_set_name = vm_set_name
```

Comment:
Why do we need `check_bridge` here? Th only caller sets it to False.

Reply:
> @wangxin Why do we need `check_bridge` here? Th only caller sets it to False.

@ZhaohuiS There're other callers with default check_bridge=True, I think Xin just let unbind don't check the bridges, to avoid error on unbinding incompleted, broken testbeds.

Reply:
The default behavior of calling the `init` method is to get list of bridges for VMs of this topology. This behavior is required for procedures like `bind` and `renumber`. For `unbind` procedure, we should skip this check to make the code more robust. That's why we need a flag `check_bridge` to distinguish the different behaviors.